README.md

```diff
@@ -18,6 +18,15 @@ This module creates the following IBM Cloud&reg; Virtual Private Cloud (VPC) net
 
 ![vpc-module](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/main/.docs/vpc-module.png)
 
+:exclamation: **[Major Version Upgrade to v8.0.0]**:
+
+This major version upgrade affects the Hub and Spoke VPC topology. The `ibm_is_vpc_dns_resolution_binding` resources are no longer created explicitly. Instead, DNS resolution bindings for Spoke VPCs are now handled within the `ibm_is_vpc` resource when the DNS resolver type is set to `delegated`.
+
+To upgrade your resources, follow this two-step process:
```

Comment:
Given the existing instruction, I suspect consumers will already have flipped the flag update_delegated_resolver to true in version 7, and this is likely still the value used. So in first step, isn't there a need to explicitly set update_delegated_resolver to false?

In addition, given the flow to delete and re-create, we should probably document network connectivity issues in dns resolution in the spoke during this process?